sdk/src/main/kotlin/io/rover/sdk/ui/blocks/image/ViewImage.kt

```diff
@@ -21,7 +21,7 @@ internal class ViewImage(
         imageView.scaleType = ImageView.ScaleType.FIT_XY
     }
 
-    override var viewModelBinding: MeasuredBindableView.Binding<ImageBlockViewModelInterface>? by ViewModelBinding(imageView) { binding, subscriptionCallback ->
+    override var viewModelBinding: MeasuredBindableView.Binding<ImageBlockViewModelInterface>? by ViewModelBinding { binding, subscriptionCallback ->
```

Comment:
I still feel like this issue can potentially occur with image blocks and background images. But if the countermeasure is too costly, then I understand leaving it off for now.